jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/ALTAdmissibleHeuristic.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2016-2018, by Dimitrios Michail and Contributors.
+ * (C) Copyright 2016-2018, by Dimitrios Michail, Semen Chudakov and Contributors.
```

Comment:
Please keep only the main author here. The rest are as "Contributors". Git blame keeps the exact logs.